<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortConfigDialog</class>
 <widget class="QDialog" name="PortConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Port Config</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Description</string>
     </property>
     <property name="buddy">
      <cstring>description</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="description"/>
   </item>
   <item>
    <widget class="QGroupBox" name="transmitModeBox">
     <property name="title">
      <string>Transmit Mode</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QRadioButton" name="sequentialStreamsButton">
        <property name="text">
         <string>Sequential Streams</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="interleavedStreamsButton">
        <property name="text">
         <string>Interleaved Streams</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Reservation</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="reservedBy">
        <property name="text">
         <string>Reserved by: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="reserveButton">
        <property name="text">
         <string>Reserve</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="testOK">
     <property name="text">
      <string>testOK</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="triggersGroup">
     <property name="title">
      <string>triggers</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="exclusiveControlButton">
     <property name="text">
      <string>Exclusive Control</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="streamStatsButton">
     <property name="text">
      <string>Stream Statistics</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>226</width>
       <height>71</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>description</tabstop>
  <tabstop>sequentialStreamsButton</tabstop>
  <tabstop>interleavedStreamsButton</tabstop>
  <tabstop>reserveButton</tabstop>
  <tabstop>exclusiveControlButton</tabstop>
  <tabstop>streamStatsButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PortConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>559</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PortConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>559</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
